src/blog.njk

```diff
@@ -82,6 +100,35 @@ permalink: "blogs/{{ blog.title | slugify }}/"
         ) }}
     {% endif %}
 
+
+    {# This can be removed once added to the CMS #}
+    {% set carouselItems = [
+        {
+            carouselContent: '<h4 class="iai-blog-carousel__author"><span>Ben Rowell</span><span> - Job title</span></h4><span class="iai-blog-carousel__text">' + "I was drawn to i.AI because I wanted to make a real difference in people's lives, and I found a team of brilliant minds who share that passion. It's exciting that I get to work with cutting-edge technology while innovating services that have the potential to positively impact people up and down the country; that combination is pretty special.</span>"
```

Comment:
For each of these people, it says `job title` instead of their actual job title

Reply:
Yeah, this was an earlier demo. I'll remove this now as it's not needed.